15-eco.Rmd

```diff
@@ -7,12 +7,19 @@ In it you will also make use of R's\index{R} interfaces to dedicated GIS\index{G
 
 The chapter uses the following packages:
 
-```{r 14-eco-1, message=FALSE, eval=FALSE}
-library(sf)
-library(raster)
-# library(RQGIS)
-library(mlr)
+```{r 15-eco-1, message=FALSE, eval=FALSE}
+library(data.table)
 library(dplyr)
+library(mlr3)
+library(mlr3spatiotempcv)
+library(mlr3tuning)
+library(mlr3learners)
+library(qgisprocess)
+library(paradox)
+library(ranger)
+library(sf)
+library(terra)
+library(tree)
 library(vegan)
```

Comment:
Do we need to load it if we're using it below with `vegan::`?

Reply:
probably not